be1-go/hub/standard_hub/mod.go

```diff
@@ -533,24 +534,24 @@ func (h *Hub) createLao(msg message.Message, laoCreate messagedata.LaoCreate,
 	laoChannelPath := rootPrefix + laoCreate.ID
 
 	if _, ok := h.channelByID[laoChannelPath]; ok {
-		return answer.NewErrorf(-3, "failed to create lao: duplicate lao path: %q", laoChannelPath)
+		return answer.NewDuplicateResourceError(fmt.Sprintf("failed to create lao: duplicate lao path: %q", laoChannelPath))
```

Comment:
This is how you should use your updated error functions:

```suggestion
		return answer.NewDuplicateResourceErrorf("failed to create lao: duplicate lao path: %q", laoChannelPath)
```